draft-ietf-taps-impl.md

```diff
@@ -367,11 +367,11 @@ An implementation can use the Capacity Profile to prefer paths that match an app
    * Low Latency/Non-Interactive:
      Prefer paths with a low expected Round Trip Time, but can tolerate delay variation;
    * Constant-Rate Streaming:
-     Prefer paths that are expected to satisfy the requested Stream Send or Stream Receive Bitrate, based on the observed maximum throughput;
+     Prefer paths that are expected to satisfy the requested stream `Send` or stream `Receive` bitrate, based on the observed maximum throughput;
```

Comment:
Not 100% sure about this one... it doesn't refer directly to the calls, it's just a specification about incoming or outgoing bitrate...

Reply:
and in line with the changes below, "send" and "receive" should not be capitalized, I believe.